xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -936,43 +948,45 @@ void CEdl::MergeShortCommBreaks()
    * Remove any spurious short commercial breaks at the very start so they don't interfere with
    * the algorithms below.
    */
-  if (!m_vecCuts.empty()
-  &&  m_vecCuts[0].action == Action::COMM_BREAK
-  && (m_vecCuts[0].end - m_vecCuts[0].start) < 5 * 1000) // 5 seconds
+  if (!m_vecEdits.empty() && m_vecEdits[0].action == Action::COMM_BREAK &&
+      (m_vecEdits[0].end - m_vecEdits[0].start) < 5 * 1000) // 5 seconds
   {
     CLog::Log(LOGDEBUG, "{} - Removing short commercial break at start [{} - {}]. <5 seconds",
-              __FUNCTION__, MillisecondsToTimeString(m_vecCuts[0].start),
-              MillisecondsToTimeString(m_vecCuts[0].end));
-    m_vecCuts.erase(m_vecCuts.begin());
+              __FUNCTION__, MillisecondsToTimeString(m_vecEdits[0].start),
+              MillisecondsToTimeString(m_vecEdits[0].end));
+    m_vecEdits.erase(m_vecEdits.begin());
   }
 
   const std::shared_ptr<CAdvancedSettings> advancedSettings = CServiceBroker::GetSettingsComponent()->GetAdvancedSettings();
   if (advancedSettings->m_bEdlMergeShortCommBreaks)
   {
-    for (int i = 0; i < static_cast<int>(m_vecCuts.size()) - 1; i++)
+    for (int i = 0; i < static_cast<int>(m_vecEdits.size()) - 1; i++)
```

Comment:
Hum because this is used to merge short commbreaks, i.e. compare two consecutive edits and if both are commbreaks and the end of the first differs only by one second w.r.t the start time of the next one - consider a single commbreak. The implementation uses index i+1 so iterating till m_vecEdits.size() would throw out of bounds.